packages/libs/error-stack/src/fmt/hook.rs

```diff
@@ -534,20 +544,20 @@ mod default {
     };
 
     #[cfg(all(nightly, feature = "std"))]
-    fn backtrace(backtrace: &Backtrace, ctx: &mut HookContext<Backtrace>) -> Emit {
+    fn backtrace(backtrace: &Backtrace, ctx: &mut HookContext<Backtrace>) -> Vec<Emit> {
```

Comment:
I don't think this should return a `Vec`, but only an `Emit`

Reply:
This is because `backtrace` (and `span_trace`) both implement the `Hook` signature. This is important in a follow-up PR, where they are default hooks instead.